project_national/housing_characteristics/Geometry Foundation Type.tsv

```diff
@@ -1,459 +1,451 @@
-7B	Multi-Family with 5+ Units	2000-09	0.000000	0.000000	0.000000	0.649138	0.350862	5.000000	295.438263	0.000027
-7B	Multi-Family with 5+ Units	2010s	0.000000	0.000000	0.000000	0.649138	0.350862	5.000000	295.438263	0.000006
-7B	Multi-Family with 5+ Units	<1940	0.321873	0.000000	0.000000	0.155580	0.522547	24.000000	3203.303467	0.000024
-7B	Single-Family Attached	1940-59	0.108427	0.505453	0.000000	0.071680	0.314440	26.000000	3624.170898	0.000010
-7B	Single-Family Attached	1960-79	0.093525	0.600123	0.000000	0.203539	0.102813	22.000000	2827.204590	0.000017
-7B	Single-Family Attached	1980-99	0.177117	0.340419	0.000000	0.339609	0.142855	13.000000	1743.880981	0.000018
-7B	Single-Family Attached	2000-09	0.000000	0.412361	0.000000	0.381459	0.206180	5.000000	295.438263	0.000010
-7B	Single-Family Attached	2010s	0.000000	0.412361	0.000000	0.381459	0.206180	5.000000	295.438263	0.000002
-7B	Single-Family Attached	<1940	0.228494	0.290110	0.000000	0.110445	0.370951	24.000000	3203.303467	0.000008
-7B	Single-Family Detached	1940-59	0.108427	0.505453	0.000000	0.071680	0.314440	26.000000	3624.170898	0.000113
-7B	Single-Family Detached	1960-79	0.089495	0.574264	0.043091	0.194768	0.098382	22.000000	2827.204590	0.000188
-7B	Single-Family Detached	1980-99	0.160421	0.308330	0.094264	0.307596	0.129389	13.000000	1743.880981	0.000198
-7B	Single-Family Detached	2000-09	0.000000	0.412361	0.000000	0.381459	0.206180	5.000000	295.438263	0.000103
-7B	Single-Family Detached	2010s	0.000000	0.412361	0.000000	0.381459	0.206180	5.000000	295.438263	0.000023
-7B	Single-Family Detached	<1940	0.228494	0.290110	0.000000	0.110445	0.370951	24.000000	3203.303467	0.000092
-# Created by: sources\recs\2009\tsv_maker.py
-# Source: The sample counts and sample weights are constructed using U.S. EIA 2009 Residential Energy Consumption Survey (RECS) microdata.
-# Assumption: All mobile homes have Pier and Beam foundations.
-# Assumption: Multi-family buildings cannot have Pier and Beam and Heated Basements
-# Assumption: Single-family attached buildings cannot have Pier and Beam foundations
-# Assumption: Foundation types are the same for each building type except mobile homes and the applicable options.
-# Assumption: Because we need to assume a foundation type for ground-floor MF units, we use the lumped SFD+SFA distributions for MF2-4 and MF5+ building foundations. (RECS data for households in MF2-4 unit buildings are not useful since we do not know which floor the unitis on. RECS does not include foundation responses for households in MF5+ unit buildings.)
-# Assumption: For SFD and SFA, if no foundation type specified, then sample has Pier and Beam foundation.
\ No newline at end of file
+Dependency=ASHRAE IECC Climate Zone 2004	Dependency=Geometry Building Type RECS	Dependency=Vintage ACS	Option=Vented Crawlspace	Option=Unvented Crawlspace	Option=Heated Basement	Option=Ambient	Option=Slab	Option=Unheated Basement	sampling_probability
```

Comment:
This change looks wrong as now we will only end up with vented crawlspaces with floor insulation (and never unvented crawlspaces with foundation wall insulation). We should be preserving the vented/unvented split that previously existed.